docs/install-new.md

```diff
@@ -0,0 +1,94 @@
+        <img src="http://azuredeploy.net/deploybutton.png"/>
+    </a>
+    <a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fakshaysngupta%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
+        <img src="http://armviz.io/visualizebutton.png"/>
+    </a>
+
+    After the deployment completes, you can look at the Output window for parameters needed in the following steps.
+
+## Setting up the Azure Kubernetes Cluster
+
+Once, we have the created the resources in Azure, we need to deploy following pods on the cluster:
+1) `aad pod identity` - This controller uses user assigned identity and provide ARM access token to the controller.
+2) `application gateway ingress controller` - This controller communicates ingress related events to the Application Gateway resource.
+
+Steps:
+
+1) Get credentials for the newly created Azure Kubernetes Cluster. This will cache the credentials in kubeconfig and set the context.  
+    ```bash
+    az aks get-credentials --resource-group <rg> --name <aksClusterName>`
+    ```
+
+2) Add aad pod identity service to the cluster using the following command. This service will be used by the controller. You can refer [aad-pod-identity](https://github.com/Azure/aad-pod-identity).  
+    ```bash
+    kubectl create -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment.yaml`
+    ```
+
+3) Install [Helm](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm) and run the following to add `application-gateway-kubernetes-ingress` helm package:
+    ```bash
+    helm init
+    helm repo add application-gateway-kubernetes-ingress https://azure.github.io/application-gateway-kubernetes-ingress/helm/
+    helm repo update
+    ```
+
+4) Edit [helm-config.yaml](example/helm-config.yaml) and fill in the values for `appgw` and `armAuth`
+    ```yaml
+    # This file contains the essential configs for the ingress controller helm chart
+
+    ################################################################################
+    # Specify which application gateway the ingress controller will manage
+    #
+    appgw:
+        subscriptionId: <subscription-id>
+        resourceGroup: <resourcegroup-name>
+        name: <applicationgateway-name>
+
+    ################################################################################
+    # Specify which kubernetes namespace the ingress controller will watch
+    # Default value is "default"
+    #
+    # kubernetes:
+    #   watchNamespace: <namespace>
+
+    ################################################################################
+    # Specify the authentication with Azure Resource Manager
+    #
+    # Two authentication methods are available:
+    # - Option 1: AAD-Pod-Identity (https://github.com/Azure/aad-pod-identity)
+    armAuth:
```

Comment:
Agree, please specify the command to use. For: eg (name has to match exactly what is created by template"
 " az identity show -g  <resource-group>  -n aaa-identity" here to get the required params below.

Also mention that "id" in the output maps to "identityResourceID" in this yaml. It is not intuitive for first time users, otherwise.

Reply:
@akshaysngupta can you provide the `az` command that can be used to retrieve the AD MSI for providing to the `Helm` config?